source/docs/hardware/hardware-basics/preemptive-troubleshooting.rst

```diff
@@ -112,11 +106,11 @@ We often see problems with the Drivers Station. You should:
 - Verify that there is nothing unnecessary running in the application tray in the start menu (bottom right side)
 
 Handy tools
-------------------------------------
+-----------
 
 .. image:: images/preemptive-troubleshooting/preCheckTools.png
```

Comment:
If this could be replaced with a non-blown out picture as well..

Reply:
Does the image I added in 5100856 work well?